bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/IndexPersistenceMgrTest.java

```diff
@@ -128,4 +130,138 @@ private void evictFileInfoTest(boolean createFile) throws Exception {
             indexPersistenceMgr.close();
         }
     }
+
+    final long lid = 1L;
+    final byte[] masterKey = "write".getBytes();
+
+    @Test(timeout = 60000)
```

Comment:
Drop explicit timeout please. Recently we choose not to use it

Reply:
Sure, I'll fix the timeout